docs/src/main/asciidoc/vertx.adoc

```diff
@@ -239,7 +239,7 @@ public class GreetingResource {
 ./mvnw compile quarkus:dev
 ----
 
-Then, open your browser to 'http://localhost:8080/lorem' and you should get the message.
+Then, open your browser to `http://localhost:8080/lorem` and you should get the message.
```

Comment:
Dunno. I think it would be better to not use anything and have a proper link that is clickable?